app/main/forms/auth_forms.py

```diff
@@ -19,9 +19,9 @@
 
 EMAIL_REGEX = r"^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$"
 EMAIL_LOGIN_HINT = "Enter the email address you used to register with the Digital Marketplace"
-PASSWORD_HINT = f"Must be between {PASSWORD_MIN_LENGTH} and {PASSWORD_MAX_LENGTH} characters"
-PASSWORD_LENGTH_ERROR_MESSAGE = f"Passwords must be between {PASSWORD_MIN_LENGTH} and {PASSWORD_MAX_LENGTH} characters"
-PASSWORD_BLACKLISTED_ERROR_MESSAGE = "Your password must be hard to guess"
+PASSWORD_HINT = f"Password must be between {PASSWORD_MIN_LENGTH} and {PASSWORD_MAX_LENGTH} characters"
+PASSWORD_LENGTH_ERROR_MESSAGE = f"Enter a password between {PASSWORD_MIN_LENGTH} and {PASSWORD_MAX_LENGTH} characters"
```

Comment:
I think having a password hint is fine, and having the error message be the same is also fine 😆 Did you find a second opinion on this anywhere?